totalRP3_Extended/aura/aura.lua

```diff
@@ -380,6 +388,14 @@ function auraCore:RemoveAura(auraId)
 	return false;
 end
 
+function auraCore:RemoveAllAuras()
+	for _, aura in EnumerateValidAuras(self.activeAuras) do
+		aura.persistent.invalid = true;
+		self:UnregisterAuraEvents(aura);
+		self:Update(true);
```

Comment:
It's safer to move the Update after the loop, otherwise somebody could use the multiple Updates to sneak in new auras while the loop runs.